test/unit/v_all/test_api.py

```diff
@@ -61,3 +61,19 @@ def test_get_bucket_by_id_v2(self):
         self.cache.save_bucket(Bucket(api=self.api, name='bucket_name', id_='bucket_id'))
         read_bucket = self.api.get_bucket_by_id('bucket_id')
         assert read_bucket.name == 'bucket_name'
+
+    @pytest.mark.apiver(from_ver=0)
```

Comment:
It is redundant

Reply:
Is consistency across the tests not better than having some tests that have the annotation and some not? Happy to change it but as someone coming in to the code I would wonder why some have and some don't

Reply:
Why marking all the tests with some defaults? The reason for having this mark is to be able to easily list/run only the tests that differ across apiver versions with e.g. `nox -s unit-3.9 -- --co -m apiver`

Reply:
I agree that we should remove it. It's clear to me that the decorator should only be used on tests which should execute for a limited subset of apiver interface versions.

Reply:
Fair enough. I already updated the PR to remove it - but sounds like it was just my confusion :)

Reply:
Already approved :)